packit/distgit.py

```diff
@@ -737,3 +737,25 @@ def get_bugzilla_ids_from_changelog(changelog: str) -> list[str]:
         bugs = [bug.group(1) for bug in re.finditer(bz_regex, changelog, re.IGNORECASE)]
         logger.debug(f"Bug IDs found in the changelog: {bugs}")
         return bugs
+
+    def get_nvr(self, dist_git_branch):
```

Comment:
There is no type info, also, I'm thinking if it makes sense for this to return [NEVR]( https://github.com/packit/specfile/blob/9aa7509bc208cd03788a233626e326003844ecd0/specfile/utils.py#L87), but that's probably unnecessary.

Reply:
added the type info, for now I would stick with the NVR that we need